cmd/thanos/query.go

```diff
@@ -317,6 +322,7 @@ func registerQuery(app *extkingpin.App) {
 			*enableTargetPartialResponse,
 			*enableMetricMetadataPartialResponse,
 			*enableExemplarPartialResponse,
+			*enableQueryExperimentalFunctions,
```

Comment:
The right way to enable experimental function is through `parser.EnableExperimentalFunctions = enablePromQLExperimentalFunctions`. It is a global var.